Add Supported Events Policy and update Reimbursement Policy

JonEsparaz opened the pull request:
##  Supported Events Policy v1.0

- Specify requirements to get a competition recognized by Speedcubing Canada, both for Directors/Officers and other individual(s) and group(s)
- Specify various organizational, financial, legal and other requirements to protect Speedcubing Canada's interests

## Reimbursement Policy v1.1

Changes as per the commit messages:

- Improve clarity (in particular, better usage of RFC 2119 terms)
- Integrate with Speedcubing Canada Supported Events Policy
- Re-word to allow for "individual(s) and/or group(s)" to request reimbursement
- Specify "audio-visual services or equipment rental" instead of just "audio-visual"
- Explicitly allow for "power bars" (in addition to "extension cords")
- Add "street cars" under "public transportation"
- Remove "Stripe Fees" from allowed expenses since Supported Events must use Speedcubing Canada's Stripe account
- Add paragraph in introduction to explicitly allow for modification of the document
- Extend allowed submission window to 365 days
- Explicitly allow other online money transfer services like PayPal
- Use proper apostrophe and quotation mark characters
- Add parking as an allowed expense

JonEsparaz committed:
Speedcubing Canada Supported Events Policy v1.0

JonEsparaz committed:
Reimbursement Policy v1.1

- Improve clarity (in particular, better usage of RFC 2119 terms)
- Integrate with Speedcubing Canada Supported Events Policy
- Re-word to allow for "individual(s) and/or group(s)" to request reimbursement
- Specify "audio-visual services or equipment rental" instead of just "audio-visual"
- Explicitly allow for "power bars" (in addition to "extension cords")
- Add "street cars" under "public transportation"
- Remove "Stripe Fees" from allowed expenses since Supported Events must
use Speedcubing Canada's Stripe account
- Add paragraph in introduction to explicitly allow for modification of the document
- Extend allowed submission window to 365 days
- Explicitly allow other online money transfer services like PayPal
- Use proper apostrophe and quotation mark characters

JonEsparaz committed:
Add parking as an allowed expense

JonEsparaz commented on public/src/reimbursement-policy.md:
```diff
@@ -2,52 +2,52 @@
-- A “Speedcubing Canada event” is defined as an event, usually an official World Cube Association competition in Canada, that is organized by or in partnership with Speedcubing Canada.
-  - Written notice from Speedcubing Canada’s Board of Directors is required to officially recognize a Speedcubing Canada event.
-  - Event organizers or WCA Delegates may email [board@speedcubingcanada.org](mailto:board@speedcubingcanada.org) to request that their event be recognized as a Speedcubing Canada event.
+- Speedcubing Canada **must** only reimburse expenses that are reasonable and necessary for running a Speedcubing Canada Supported Event as defined by the [Speedcubing Canada Supported Events Policy](https://www.speedcubingcanada.org/documents/supported-events-policy.pdf).
```
The link obviously won't work until I publish the documents.

AlyssaHope commented on public/src/reimbursement-policy.md:
```diff
@@ -2,52 +2,52 @@
+
+Individual(s) and/or group(s) **must not** expect reimbursement from Speedcubing Canada, even if the expense is covered by the policies described in this document. If individual(s) and/or group(s) are planning to request reimbursement for allowed expenses, they _should_ discuss the expense with Speedcubing Canada’s Board of Directors prior to making any out-of-pocket payment(s).
+
+Speedcubing Canada’s Board of Directors reserves the right to modify this document (including the additional or removal of allowed expenses) with a majority of not less than two-thirds (2/3) of Speedcubing Canada’s Board of Directors vote in favour of the modification.
```
```suggestion
Speedcubing Canada’s Board of Directors reserves the right to modify this document (including the addition or removal of allowed expenses) with a majority of not less than two-thirds (2/3) of Speedcubing Canada’s Board of Directors vote in favour of the modification.
```

AlyssaHope reviewed (changes requested):
typo fix :)

JonEsparaz committed:
Fix typo: "addition" vs "additional"

Co-authored-by: AlyssaHope <67570875+AlyssaHope@users.noreply.github.com>

kr-matthews commented on public/src/supported-events-policy.md:
```diff
@@ -0,0 +1,48 @@
+
+All dollar amounts discussed are in Canadian dollars.
+
+Speedcubing Canada’s Board of Directors reserves the right tomodify this document with a majority of not less than two-thirds (2/3) of Speedcubing Canada’s Board of Directors vote in favour of the modification.
```
"tomodify"

bmetherall replied:
```suggestion
Speedcubing Canada’s Board of Directors reserves the right to modify this document if a majority of not less than two-thirds (2/3) of Speedcubing Canada’s Board of Directors vote in favour of the modification.
```

bmetherall commented on public/src/supported-events-policy.md:
```diff
@@ -0,0 +1,48 @@
+- Written notice from Speedcubing Canada’s Board of Directors **must** be obtained to officially recognize a Speedcubing Canada Supported Event.
+- Individuals seeking to run a Speedcubing Canada Supported Event **must** email [board@speedcubingcanada.org](mailto:board@speedcubingcanada.org) to request that their event be recognized as a Speedcubing Canada Supported Event.
+  - The email **must** include a detailed, itemized budget for the event, including all expenses and sources of income.
+    - The budget _should_ be balanced and _should_ include a not insignificant margin to account for unexpected costs and circumstances.
```
Do you have a particular amount or percentage for the "not insignificant margin"?

bmetherall commented on public/src/supported-events-policy.md:
```diff
@@ -0,0 +1,48 @@
+## Other
+
+- Speedcubing Canada’s Board of Directors reserves the right to reject recognition of an event as a Speedcubing Canada Supported Event.
+- Speedcubing Canada’s Board of Directors reserves the right to ban specific individual(s) and/or group(s) who do not comply with Speedcubing Canada’s By-Laws and/or Policies from running Speedcubing Canada Supported Events for a fixed period of time or indefinitely.
```
```suggestion
- Speedcubing Canada’s Board of Directors reserves the right to ban specific individual(s) and/or group(s) who do not comply with [Speedcubing Canada’s By-Laws](https://www.speedcubingcanada.org/documents/by-laws.pdf) and/or Policies from running Speedcubing Canada Supported Events for a fixed period of time or indefinitely.
```
Add link to by-laws.

JonEsparaz committed:
Improve clarity, fix typos, add link to by-laws

JonEsparaz committed:
Change "updated" date of policies

JonEsparaz committed:
Re-organize Equipment policy into six sub-types

JonEsparaz commented on public/src/reimbursement-policy.md:
```diff
@@ -49,43 +49,57 @@ Speedcubing Canada’s Board of Directors reserves the right to modify this docu
 - Reimbursement requests are allowed to be made for equipment used for a Speedcubing Canada Supported Event.
 
-#### Type 1
+#### Type 1: timing equipment
```
@kr-matthews interested in your feedback on the re-organization of this section. Thanks!

kr-matthews replied:
It looks a lot clearer, thanks.
Is there a difference between reimbursement for equipment for a particular supported event vs for general equipment to be used at all supported events?

JonEsparaz replied:
For now, I would just name a recent or upcoming Supported Event to associate the cost with (I assume we're buying stuff to eventually use at an event).

In the future, I'd like to have a way to request equipment/reimbursement for general use as you described, but that's outside the scope of this round of edits.

kr-matthews replied:
Including major expenses (like a laptop or printer) in a particular competition makes it hard to balance the budget, but having them separate and keeping all budgets (roughly) balanced doesn't seem sustainable mathematically.

JonEsparaz replied:
We are tracking general equipment purchases (for items procured directly by Speedcubing Canada) under a separate, non-competition budget.

Let me think about how we can better support "general equipment" in this policy...

JonEsparaz replied:
I added some flexibility for "general equipment" expenses: https://github.com/Speedcubing-Canada/docs-speedcubing-canada/pull/4/commits/feb8239cca939bdaea134b4c3f52ebdca2e130dd.

Still want to maintain some rigidity - let me know if you think this works.

kr-matthews replied:
I think this is good, thanks. My only theoretical concern would be regions consistently running balanced budgets, but including lots of expenses under "general equipment" and therefore constantly losing money. I don't think it's a practical concern at this point though.

kr-matthews commented on public/src/reimbursement-policy.md:
```diff
@@ -2,107 +2,121 @@
+- Expenses covered by this policy include:
   - Cube covers
-  - Stationary (e.g. pens, markers)
   - Tri-folds
```
what about pipe and drapes?

JonEsparaz replied:
Added

kr-matthews commented on public/src/reimbursement-policy.md:
```diff
@@ -2,107 +2,121 @@
+  - Notepads
   - Stationary (e.g. pens, markers, highlighters, binder clips, clipboards, labels, tape, white-out)
-  - Storage (e.g. baskets, bins, boxes, pencil cases)
+  - Storage devices (e.g. baskets, bins, boxes, pencil cases)
```
suitcases? we use two, and are planning on requesting another one (or two?)

JonEsparaz replied:
The `e.g.` isn't meant to be an exhaustive list, but I'll add it 👍🏼

JonEsparaz committed:
Add pipe and drape as scrambling station blockers, suitcases as storage devices

JonEsparaz committed:
Allow for "general equipment" expenses, pluralize Speedcubing Canada Supported Events

https://github.com/Speedcubing-Canada/docs-speedcubing-canada/pull/4#discussion_r929473722

JonEsparaz committed:
Improve clarity, fix typo

bmetherall commented on public/src/reimbursement-policy.md:
```diff
@@ -114,47 +128,51 @@ All dollar amounts discussed are in Canadian dollars.
   - For each receipted expense, the email **must** include which policy (e.g. Venues and associated costs, Equipment Type 1) the expense is covered by.
-- Reimbursements _should_ be submitted within 60 days of when the out-of-pocket payment was made and **must** be submitted within 90 days of when the out-of-pocket payment was made.
-  - Reimbursements submitted between 61 and 90 days of when the out-of-pocket payment was made **must** include a written explanation for the delay.
+- Reimbursements _should_ be submitted within 60 days of when the out-of-pocket payment was made and **must** be submitted within 365 days of when the out-of-pocket payment was made.
```
I don't have a strong opinion on this, but a year feels excessive and potentially annoying. I think 180 days is more realistic. Perhaps we should have a quick poll of the officers?

bmetherall replied:
No one seems to have an opinion.

bmetherall commented on public/src/reimbursement-policy.md:
```diff
@@ -2,107 +2,121 @@
+  - Personal protective equipment (e.g. medical masks, gloves)
+  - Cleaning/sanitary products (e.g. wipes, hand sanitizer, cling wrap)
+
+#### Type 6: other
```
I don't really understand the distinction between type 2 and type 6. Is there a reason they are not combined?

bmetherall commented on public/src/reimbursement-policy.md:
```diff
@@ -2,107 +2,121 @@
-  - Professional printing (e.g. programs, banners, table runners, stickers)
-  - Cardstock and paper (e.g. for name tags, schedules or certificates)
+  - Cardstock and paper
+  - Professional printing (e.g. banners, table cloths, table runners, print programmes, signs, stickers, name badges)
```
I think 'program' is more common in Canadian English
[Canadian Oxford Dictionary](https://www.oxfordreference.com/view/10.1093/acref/9780195418163.001.0001/acref-9780195418163?avail_01=free&avail_02=unlocked&btog=chap&hide=true&jumpTo=program&page=2762&pageSize=20&skipEditions=true&sort=titlesort&source=%2F10.1093%2Facref%2F9780195418163.001.0001%2Facref-9780195418163)

JonEsparaz replied:
fixed

bmetherall commented on public/src/supported-events-policy.md:
```diff
@@ -0,0 +1,48 @@
+  - The email **must** include a detailed, itemized budget for the event, including all expenses and sources of income.
+    - The budget _should_ be balanced and _should_ include a margin of at least 10% to account for unexpected costs and circumstances.
+  - If the email request is made by a Speedcubing Canada Director or Officer, the event will be recognized as a Speedcubing Canada Supported Event unless a Director raises an objection within 72 hours of the request.
+    - If an objection is raised by a Director, the event will be recognized as a Speedcubing Canada Supported Event if a majority of not less than two-thirds (2/3) of Speedcubing Canada’s Board of Directors vote in favour of recognition.
```
If a director requesting recognition, do they still get to vote?

JonEsparaz replied:
Probably makes sense to exclude them. I don't expect this to happen very often.

bmetherall commented on public/src/supported-events-policy.md:
```diff
@@ -0,0 +1,48 @@
+
+- Any and all Speedcubing Canada Supported Events **must** follow all legal requirements of the jurisdiction in which in the event takes place.
+- Any and all Speedcubing Canada Supported Events **must** be covered by a liability insurance policy (either provided by Speedcubing Canada directly or purchased by another party) with not less than $1 million coverage that names Speedcubing Canada as an insured party.
+  - If the insurance policy is not provided by Speedcubing Canada directly, proof of insurance **must** be emailed to [board@speedcubingcanada.org](mailto:board@speedcubingcanada.org).
```
```suggestion
  - If the insurance policy is not provided by Speedcubing Canada directly, proof of insurance **must** be emailed to [board@speedcubingcanada.org](mailto:board@speedcubingcanada.org) before their event is recognized as a Speedcubing Canada Supported Event.
```
Or a similar qualification.

JonEsparaz replied:
In most cases Speedcubing Canada will purchase the insurance directly or the organizer will purchase the insurance via the venue (most municipalities in Ontario offer this).

I am comfortable with the insurance from the latter case being submitted "not less than two weeks before the first date of the event". If they don't provide proof of insurance, we can revoke recognition, or the more reasonable option would be purchasing ourselves at that point.

Anyways, hopefully this won't be a long term problem once we have ongoing coverage.

bmetherall commented on public/src/reimbursement-policy.md:
```diff
@@ -2,107 +2,121 @@
-  - O-rings
+  - O-rings and other button protectors
+
+#### Type 2: non-timing equipment
```
Pedantic point: I don't like calling this 'non-timing equipment' since that's everything that isn't type 1.

bmetherall commented on public/src/reimbursement-policy.md:
```diff
@@ -2,107 +2,121 @@
+Version: 1.1
+Updated: July 25, 2022
 
 # Speedcubing Canada Reimbursement Policy
```
I assume this isn't supposed to be exhaustive. We should add something that allows someone to provide an explanation for an item not covered by this policy.

JonEsparaz replied:
I would prefer we explicitly add items, which is possible, but intentionally takes a bit of effort. An open ended option is too flexible in my opinion.

Are there specific items you think we're missing?

bmetherall replied:
I didn't really have anything in mind at the time. Reading through the NAC financial report we haven't explicitly included radios, dollies, number pads, or Square-1 tabs. They are covered by either tools, laptop accessories, or printing, but it may be worth adding them.

Re: open ended being too flexible, I'm okay with that if that's your intention.

bmetherall commented on public/src/reimbursement-policy.md:
```diff
@@ -2,107 +2,121 @@
-  - Harmonica holders (only if necessary for 4x4x4 Blindfolded, 5x5x5 Blindfolded or 3x3x3 Multi-Blind)
-  - Cardstock and paper (e.g. for scorecards, FMC, 3x3x3 Blindfolded, 4x4x4 Blindfolded, 5x5x5 Blindfolded or 3x3x3 Multi-Blind)
+  - Scrambling station blockers (e.g. tri-folds, pipe and drape)
+  - Blind blockers and harmonica holders
```
```suggestion
  - Sight blockers and harmonica holders
```
Term used in the [regs](https://www.worldcubeassociation.org/regulations/#B4c).

JonEsparaz replied:
Fixed

JonEsparaz committed:
Additional requirements, re-organize equipment (again), fix typos

JonEsparaz committed:
Bump dates, generate PDFs
